packages/jellyfish-api-core/src/category/spv.ts

```diff
@@ -156,6 +156,55 @@ export class Spv {
       { cost: 'bignumber', estimatedReward: 'bignumber' }
     )
   }
+
+  /**
+   * List anchors
+   *
+   * @param {ListAnchorsOptions} [options]
+   * @param {number} [options.minBtcHeight=-1]
+   * @param {number} [options.maxBtcHeight=-1]
+   * @param {number} [options.minConfs=-1]
+   * @param {number} [options.maxConfs=-1]
+   * @return {Promise<ListAnchorsResult[]>}
+   */
+  async listAnchors (
+    options: ListAnchorsOptions = { minBtcHeight: -1, maxBtcHeight: -1, minConfs: -1, maxConfs: -1 }
```

Comment:
```suggestion
    options: ListAnchorsOptions = { }
```

since you are going to provide the default in line 173, it will be redundant to declare them here